__test__/fts.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { searchQuery, SearchQuery } from '../src';
+
+const maybe = process.env.CI ? test.skip : test;
```

Comment:
Does this mean we skip these tests on CI? Is that due to the cb server version not supporting FTS?

Reply:
Tests were marked to be skipped in the CI due to the data being stored in an FTS index and we don't know the time for they to be ready and return the correct values, so we tried adding some delays but it is uncertain and only worked a few times with 15 sec of delay which is too much delay and even with that fails most the times.

Note: Testing indexed data in a CI can be tricky